python/ray/data/_internal/stage_impl.py

```diff
@@ -88,7 +88,9 @@ def do_randomize(self, block_list, *_):
 class RandomShuffleStage(AllToAllStage):
     """Implementation of `Dataset.random_shuffle()`."""
 
-    def __init__(self, seed: Optional[int], output_num_blocks: Optional[int]):
+    def __init__(
+        self, seed: Optional[int], output_num_blocks: Optional[int], **ray_remote_args
```

Comment:
Here it should already be `remote_args: Optional[Dict[str, Any]] = None`

Reply:
Done.